api/utils/utils.py

```diff
@@ -0,0 +1,16 @@
+def verify_data(Tx, data):
+    import requests
+    import json
+    import hashlib
+    import base64
+    r = requests.get(
+        'http://localhost:5000/get_transaction/'+Tx)
+    message = r.json()
+    json_data = json.dumps(data).encode('utf-8')
+    hash_data = hashlib.sha256(
+        json_data).hexdigest().encode()
```

Comment:
Why there is a newline? you can use `flake8` or `yapf` to format the code

Reply:
VS code auto format

Reply:
Fix in f01cf8f.